proto/services/user_svc/v1/organization_svc.proto

```diff
@@ -27,22 +27,6 @@ message CreateOrganizationResponse {
   string id = 1;
 }
 
-//
-// CreateOrganizationForUser
-//
-
-message CreateOrganizationForUserRequest {
```

Comment:
and how are we going to create non-personal orgs?

Reply:
https://buf.build/helpwave/services/docs/main:services.user_svc.v1#services.user_svc.v1.OrganizationService.CreateOrganization